labs/computing-experiment-metrics/metadata.md

```diff
@@ -2,7 +2,7 @@
 title: Computing metrics with event-level experiment data
```

Comment:
Feedback left online about the lab content. Reproduced here.

Reply:
“We’ll start by download” => “We’ll start by downloading”

Reply:
Which Optimizely project does the API token belong to? Are we comfortable with this token being public? Does it have ability to change Optimizely data for that project? In which case, we may not want to make it public.

Reply:
I like the story behind the lab, but it does feel a bit like a cliff-hanger (but I imagine that’s partly by design because this is part 1 of a series?)

Reply:
Did you mean for # !oevents load --type decisions --experiment 18786493712 --date 2020-09-14 to look like a comment in the lab? (I think so because it looks like you coded it up so that theoretically you could use the notebook to run that command).

Reply:
Did you mean to include enriching_decision_data.ipnyb  in the lab directory?

Reply:
Thank you!  The token is scope to read-only access.  I actually just replaced it with a new read-only token associated with an anonymous email address.  Great feedback; thank you--I made some changes to address and left additional comments in slack.